draft/2021-11-10-this-week-in-rust.md

```diff
@@ -18,17 +18,44 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 ### Foundation
 
+* [Launching the Rust Foundation Quarterly AMAs: Join the Conversation](https://foundation.rust-lang.org/posts/2021-11-04-rust-foundation-ama-launch/)
+
 ### Project/Tooling Updates
 
 * [SixtyFPS (GUI crate): Changelog for 7th of November 2021](https://sixtyfps.io/thisweek/2021-11-08.html)
+* [Rust Analyzer Changelog #102](https://rust-analyzer.github.io/thisweek/2021/11/08/changelog-102.html)
+* [Intellij Rust Changelog #159](https://intellij-rust.github.io/2021/11/08/changelog-159.html)
 
 ### Research Papers
 
+### Newsletter
+
+* [This Month in Rust OSDev (October 2021)](https://rust-osdev.com/this-month/2021-10/)
+* [This Month in Rust GameDev #27](https://gamedev.rs/news/027/)
+
 ### Observations/Thoughts
 
+* [View types for Rust](https://smallcultfollowing.com/babysteps//blog/2021/11/05/view-types/)
+* [What Memory Model Should the Rust Language Use?](https://paulmck.livejournal.com/66175.html)
+* [C++ Move Semantics Considered Harmful (Rust is better)](https://www.thecodedmessage.com/posts/cpp-move/)
+* [It's Time to Get Hyped About Const Generics in Rust](https://nora.codes/post/its-time-to-get-hyped-about-const-generics-in-rust/)
+
 ### Rust Walkthroughs
 
-* [series] [video] [Rust Book Club #5: Structs!](https://www.youtube.com/watch?v=mBi_FsPKd9w)
+* [Working with signals in Rust](https://www.jameselford.com/blog/working-with-signals-in-rust-pt1-whats-a-signal/)
```

Comment:
This is a duplicate, was in issue 415 also.

Reply:
Good catch! Removed